corehq/apps/cloudcare/tests/test_session.py

```diff
@@ -54,10 +54,13 @@ def test_user_data_profile(self):
         definition.save()
         profile = CustomDataFieldsProfile(name='prof', fields={'word': 'supernova'}, definition=definition)
         profile.save()
-        user = CommCareUser(
-            domain='cloudcare-tests',
-            username='worker@cloudcare-tests.commcarehq.org',
-            _id=uuid.uuid4().hex,
+        user = CommCareUser.create(
+            'cloudcare-tests',
```

Comment:
nit: it would be nice for these arguments to be named (I have no clue what the `None`s are) although this definitely shouldnt block

Reply:
agreed, but calling this out of scope for this PR